src/software/SfM/CMakeLists.txt

```diff
@@ -338,7 +338,9 @@ IF(OpenMVG_USE_OPENCV)
     vlsift
     ${OpenCV_LIBS})
   target_include_directories(openMVG_main_ComputeFeatures_OpenCV PRIVATE ${OpenCV_INCLUDE_DIRS})
-  target_compile_definitions(openMVG_main_ComputeFeatures PRIVATE USE_OCVSIFT)
+  IF(OpenMVG_USE_OCVSIFT)
+    target_compile_definitions(openMVG_main_ComputeFeatures_OpenCV PRIVATE USE_OCVSIFT=1)
```

Comment:
Does the =1 is required?
`target_compile_definitions(openMVG_main_ComputeFeatures_OpenCV PRIVATE USE_OCVSIFT)` should work

Reply:
@tomas789 Can you test with just `target_compile_definitions(openMVG_main_ComputeFeatures_OpenCV PRIVATE USE_OCVSIFT)`